LevelComponents/Entity.cpp

```diff
@@ -255,6 +255,100 @@ namespace LevelComponents
         return pixmap.toImage();
     }
 
+    /// <summary>
+    /// Render oam array in test mode.
+    /// </summary>
+    /// <param name="oamNum">
+    /// the number of oam.
+    /// </param>
+    /// <param name="nakedOAMarray">
+    /// Naked OAM array without oam number as the first element
+    /// </param>
+    /// <param name="noReferrneceBox">
+    /// Don't draw the yellow referrence Box if set this to true
+    /// </param>
+    QImage Entity::TestRenderOams(int oamNum, unsigned short *nakedOAMarray, bool noReferrneceBox)
```

Comment:
The function has test in the name. Is it used for testing? Or something else?

If it's used for testing, we should not put it in this file. If it's not used for testing, we should change the name.

Reply:
replaced with RenderOAMPreview